clusterloader2/pkg/measurement/common/network-policy/manifests/clusterrole.yaml

```diff
@@ -0,0 +1,11 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{.Name}}
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["networking.k8s.io"]
+  resources: ["networkpolicies"]
+  verbs: ["get", "list", "watch"]
```

Comment:
Nitpick: `"watch"` for networkpolicies isn't required, is it?

Reply:
Apparently neither `"list"` is being used. I'm removing both `"list"` and `"watch"` for network policies.
They are being used by the test client app https://github.com/kubernetes/perf-tests/tree/master/network/tools/network-policy-enforcement-latency